DEPLOYING.md

```diff
@@ -1,62 +1,351 @@
+```
 
-### Source files and configuration
+Hit `Enter` on every question.
 
-Now we have the runtime, next we need to download the cluster configuration files and build the services.
+5. Run
+```
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml up -d --build nginx
+```
 
-Navigate to where you want the source files to live on your server (e.g. `/opt`) and clone the repository:
+and wait setup to finish
 
+6. Run
 ```
-cd /opt
-git clone https://github.com/ViderumGlobal/ckan-cloud-docker.git
-cd ckan-cloud-docker
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml down
+``` 
+
+and wait to finish.
+
+7. In the folowing files change `CKAN_BRANCH:-ckan-2.7.3` to `CKAN_BRANCH:-ckan-2.8.1`:
+  * **`docker-compose.yaml`** - line #103
+  * **`.docker-compose.vital-strategies-philippines-theme.yaml`** - lines #24 and #43
+  * **`ckan/Dockerfile`** - line #61
+
+8. Run
+```
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml up -d --build nginx
+```
+
+again and wait to finish and get prompt in terminal.
+
+10. Setup should be finished successfully. You can check logs
+```
+docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-philippines-theme.yaml logs -f
 ```
 
-#### Environment variables
-To create or update files with secrets and env vars run and follow all steps:
+and if there is no errors CKAN instance is installed. To exit logs press `CTRL+C`
+
+11. Now we should create sysadmin user
 ```
-make secret
+docker-compose exec ckan ckan-paster --plugin=ckan sysadmin add -c /etc/ckan/production.ini admin password=12345678 email=admin@localhost
 ```
 
-#### Traefik proxy service
+Now we can access our CKAN server by visiting `localhost:8080` in browser inside Ubuntu VM.
+Login to CKAN at http://localhost:8080 with username `admin` and password `12345678`.
+You should be able to see
+
+![](https://i.imgur.com/PSizKmS.png)
+
+---
+---
+
+## 6. Exposing CKAN Vital Strategies to outside World
+
+Now, it's time to make our CKAN instance to be visible from host Windows Server 2008.
+There are two ways for doing this:
+1. using port forwarding
+2. using `bridge adapter` in network configuration
+
+### 1. Port forwarding method
+
+In Ubuntu VM settings go to network section and for the first adapter
+
+![](https://i.imgur.com/R88FkM1.png)
+
+click on `Advanced` than `Port Forwarding`, and add new rule (here `Rule 1`).
+
+![](https://i.imgur.com/z3ERRzF.png)
+
+In VirtualBox go to `File --> Host Network Manager`
+
+![](https://i.imgur.com/ZRJBMsw.png)
+
+Here we should noticed that ip address of our Ubuntu VM in this case is `192.168.56.1`.
+**Now we can access our CKAN server from host Windows Server 2008 by visiting `192.168.56.1:8080`**
+
+### 2. Using `bridge adapter` method
```

Comment:
Let's pick one of them... This one's suggested by Adam as much as I remember so let's keep this